### ISSUE TITLE
Update babel to use present-env and browserlist

### ISSUE THREAD

**Issue by jackmcpickle** [aligent/mage2themebuilder#8]
Current we use an old browser list by default. 
```
browsers: [
            'Android 2.3',
            'Android >= 4',
            'Chrome >= 20',
            'Firefox >= 24',
            'Explorer >= 8',
            'iOS >= 6',
            'Opera >= 12',
            'Safari >= 6'
        ]
```

should we move to update this and reduce our CSS build? One that I have used in the past is `[">0.25% in AU", "not ie 9", "not op_mini all"]`

IE this would kill IE10 in Aus as it currently sits at 0.03% of browsers for the flexbox prop. 

Also using babel-present-env allows us to pass this browserList to babel - which is the new standard way, as most es2015 features are now supported. 

Thoughts?

**Comment by jackmcpickle** [comment]
@luke-denton-aligent  sorry, raised an issue first before I implement.

**Comment by luke-denton-aligent** [comment]
@jackmcpickle I agree, at the very least, we shouldn't be bothering with targeting IE8, 9 or 10. Feel free to make an update to this as you would like. Could you also review the frontend-toolkit to see if a similar improvement could be implemented
